android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -1236,6 +1281,40 @@ public void run() {
     }, delay);
   }
 
+  protected void showHelpBubble() {
+    String hintText = context.getString(R.string.help_bubble_text);
+
+    // signalHelpBubbleDismissal - defined in android-host.js, gives a helpBubbleDismissed signal.
+
+    // TODO:  ensure it's called against the 'main' thread.
+    loadJavascript("keyman.showGlobeHint(\"" + hintText + "\", signalHelpBubbleDismissal);");
```

Comment:
If `hintText` has a `"` or `\` in it, things are gonna get sad real quick. We can use a JSON encoder to get a safe string to blat out here.